forte/data/data_pack.py

```diff
@@ -244,7 +243,7 @@ def text(self) -> str:
     @property
     def audio(self) -> Optional[np.ndarray]:
         r"""Return the audio of the data pack"""
-        return self.get_payload_data_at(Modality.Audio, 0)
+        return cast(np.ndarray, self.get_payload_data_at(Modality.Audio, 0))
```

Comment:
I think this `cast` is not safe, not everything can be cast this way

Reply:
There was a  typing issue `mypy forte`, and I haven't find a better way to solve it.

Reply:
Actually, I don't think we should assume `audio` to be `ndarray`